lib/Doctrine/DBAL/Schema/Expressions/Expression.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Doctrine\DBAL\Schema\Expressions;
+
+class Expression
+{
+    /** @var mixed $value */
+    protected $value;
+
+    /**
+     * @param mixed $value
+     */
+    public function __construct($value)
+    {
+        $this->value = $value;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getValue()
+    {
+        return $this->value;
+    }
+
+    public function __toString() : string
```

Comment:
Let's avoid magic and declare it as `toString()` which will need to be called explicitly.

Reply:
fixed